srcpkgs/font-manjari/template

```diff
@@ -21,6 +19,4 @@ do_install() {
 
 	# install fontconfig file
 	vinstall 67-fonts-smc-manjari.conf 0644 etc/fonts/conf.avail/67-fonts-smc-manjari.conf
-
-	vlicense LICENSE.txt
```

Comment:
They moved the license to another file https://gitlab.com/smc/fonts/manjari/-/blob/master/OFL.txt , but I guess it didn't need to be included at all? It *does* have a customized copyright line...

Reply:
AFAICT the license is OFL so it does not need to be included :)